### PR TITLE
Mp lxai

### PR DESCRIPTION
**Pull request by mpantoja314**
Sorted poster presenters
added new sponsors
added link to registration for workshop

### CODE REVIEW

File: _includes/neurips/2019/about.neurips.2019.html [hunk 1]
```diff
@@ -97,13 +98,35 @@ <h3>Platinum</h3>
 		  <a target="_blank" href="https://www.google.com"><img width=500
```
Comment (quickresolve) [inline _includes/neurips/2019/about.neurips.2019.html]: - Please add missing sponsor Facebook under platinum.
- Deepmind should be listed under Silver, not Gold.

File: _includes/neurips/2019/about.neurips.2019.html [hunk 2]
```diff
@@ -26,8 +26,9 @@ <h3>Important Dates</h3>
               <s><li>October 3rd, 2019: <a target='_blank' href="https://docs.google.com/forms/d/e/1FAIpQLSeMxJE96MyQ9KLFVX2HlB7TqLTCqhzTU_eUpz8pOwd6h8dh5g/viewform?usp=sf_link">LXAI Travel grant application</a> deadline</li></s>
               <s><li>October 3rd, 2019: <a target='_blank' href="https://nips.cc/TravelApplication">NeurIPS Travel grant application</a> deadline</li></s>
               <s><li>October 4th, 2019: Recommended deadline for <a target='_blank' href="http://www.cic.gc.ca/english/information/applications/visa.asp">Visa application</a></li></s>
-              <li>October 23rd, 2019: NeurIPS early registration deadline</li>
-              <li> November 11th, 2019: <a target='_blank' href="#">NeurIPS Child Care Registration deadline</a></li>
+              <li>October 23rd, 2019: NeurIPS early registration deadline</li> 
+	      <li>Nov 1st, 2019: Registration starts for LXAI<a target='_blank' href="https://www.eventbrite.com/publish?crumb=311f3f067b6a32&eid=77877199801">Workshop</a></li>
```
[the remark this answers is not in the document]
Comment (mpantoja314) [reply _includes/neurips/2019/about.neurips.2019.html]: done

File: _includes/neurips/2019/about.neurips.2019.html [hunk 3]
```diff
@@ -26,8 +26,9 @@ <h3>Important Dates</h3>
...
+	      <li> November 11th, 2019: <a target='_blank' href="#">NeurIPS Child Care Registration deadline</a></li>
```
[the remark this answers is not in the document]
Comment (mpantoja314) [reply _includes/neurips/2019/about.neurips.2019.html]: done

File: _includes/neurips/2019/about.neurips.2019.html (hunk 1 above)
Comment (mpantoja314) [reply _includes/neurips/2019/about.neurips.2019.html]: I did the images to the nips folder

**Comment by mpantoja314** [comment]
I did move images from sponsors to correct folder
added the correct link for eventbrite and kids 
Corrected the sponsors list and added a tag for presenters but I still don't know the order of presentation so I'm leaving that one blank. If you want I can add some random number there until I get the correct one

**Comment by mpantoja314** [comment]
I did add the presenters order
I did put all logos under 
assets/img/NIPS/2019/sponsors
and the images for the presenters on
assets/img/NIPS/2019/speakers

**Pull request merged**